xinetd/config.sls

```diff
@@ -0,0 +1,16 @@
+{%- from "xinetd/map.jinja" import xinetd with context -%}
+
+{%- if xinetd.services is iterable %}
+{%- for service, config in xinetd.services.iteritems() %}
+xinetd_{{ service }}_config:
+  file.managed:
+    - name: /etc/xinetd.d/{{ service }}
+    - source: salt://xinetd/files/config_template
+    - template: jinja
+    - context:
+        service: {{ service }}
+        config: {{ config|tojson }}
```

Comment:
```suggestion
        config: {{ config|json }}
```

Unfortunately, we can't use `|tojson` just yet since we need to support `2017.7` for the time being.  For more information about this, please refer back to https://github.com/saltstack-formulas/consul-formula/pull/40#issuecomment-472728517 and the linked content.

Reply:
There's also a second issue about using `config` here: https://github.com/saltstack-formulas/ufw-formula/pull/7#discussion_r284164796.